routes/web.php

```diff
@@ -467,12 +467,12 @@
     Route::get(
         'logout',
         [LoginController::class, 'logout']
-    )->name('logout');
+    )->name('logout.get');
```

Comment:
If logouts work with this new change, I’m all for it. I’m worried, though, that I should see any form-wise references to a logout route should star to break with this?

My inclination would be to have ‘logout’ (for the POST route, maybe?) and ‘logout.get’) for the GET route. Or, maybe vice-versa - ‘logout’ for the GET route, ‘logout.post’ for the POST route.

But still, so long as Logout actually works, I’m fine with it!

Reply:
It's already updated in all of the blades :)